src/components/molecules/AnnouncementMessage/AnnouncementMessage.tsx

```diff
@@ -14,42 +14,42 @@ import { LinkButton } from "components/atoms/LinkButton";
 import "./AnnouncementMessage.scss";
 
 export interface AnnouncementMessageProps {
-  banner?: Banner;
   isAnnouncementUserView?: boolean;
 }
 
 export const AnnouncementMessage: React.FC<AnnouncementMessageProps> = ({
-  banner,
   isAnnouncementUserView = false,
 }) => {
   const {
-    isShown: isAnnouncementMessageVisible,
```

Comment:
if we don't use `isShown` state, do we need that `useShownHide` hook at all?

Reply:
restored isShown, removed by mistake a4e73af